React-frontend/src/pages/admin/AddUser.js

```diff
@@ -0,0 +1,73 @@
+import React, { useReducer } from 'react';
+import { MDBBtn, MDBCol, MDBContainer, MDBInput } from 'mdbreact';
+import { useDispatch, useSelector } from 'react-redux';
+
+import formReducer from '../../utils/formReducer';
+import { addUser } from '../../store/actions/users';
+import Dropdown from '../../components/core/Dropdown';
+
+const AddUser = ({ history }) => {
+  const dispatch = useDispatch();
+  const initialFormData = {
+    name: '',
+    email: '',
+    password: '',
+    role: 'admin',
+  };
```

Comment:
Hey hi @amlannandy , great efforts, but why did you use role "admin" as a defualt role for an user, I think the roles of the users could be extractor or management ?

Reply:
Hey! No specific reason for that. Admin is the first option in the dropdown menu. So I just put it as default. And thanks for pointing out the roles error! I only used the admin and user roles for testing, but forgot to change them later. Fixed them now. 👍🏻